src/controllers/applications.ts

```diff
@@ -0,0 +1,38 @@
+import { ApplicationService } from "../service/application-service";
+import { ApplicationParams } from "../util/types";
+
+export class Applications {
+  async createApplication(applicationParams: ApplicationParams) {
+    try {
+      return await new ApplicationService().createApplication(
```

Comment:
No need for 'await' when only returning a value. The return will wait for the reponse.